ardcvocabs/src/main/java/au/org/aodn/ardcvocabs/service/ArdcVocabServiceImpl.java

```diff
@@ -26,9 +30,60 @@ public class ArdcVocabServiceImpl implements ArdcVocabService {
     protected RestTemplate restTemplate;
     protected RetryTemplate retryTemplate;
 
-    protected Function<JsonNode, String> label = (node) -> node.has("prefLabel") ? node.get("prefLabel").get("_value").asText() : null;
-    protected Function<JsonNode, String> about = (node) -> node.has("_about") ? node.get("_about").asText() : null;
-    protected Function<JsonNode, String> definition = (node) -> node.has("definition") ? node.get("definition").asText() : null;
+    protected Function<JsonNode, String> extractSingleText(String key) {
+        return (node) -> {
+            JsonNode labelNode = node.get(key);
+            if (labelNode != null) {
+                if (labelNode.has("_value")) {
+                    return labelNode.get("_value").asText();
+                }
+                if (labelNode instanceof TextNode) {
+                    return labelNode.asText();
+                }
+                return null;
```

Comment:
This line no need?

Reply:
I think needed, there is case that the node is just a plain text, where also case it is a json-like

E.g node = "test" vs node = {"_value": "sample"}

@utas-raymond

Reply:
In particular, that line for textNode stuff helps to extract "displayLabel" , without it, can't get 'displayLabel' like with "prefLabel" you previous familiar with, no "_value" field.